src/Controller/AdminController.php

```diff
@@ -135,21 +138,41 @@ public function showUser(int $id): void
     /**
      * @param int $id
      *
+     * @throws ReflectionException
      * @throws Exception
      */
     public function editUser(int $id): void
```

Comment:
Reformat with only on render. the condition is if the form is submit and valid then...